src/coreclr/vm/codeman.cpp

```diff
@@ -3532,7 +3661,52 @@ void EEJitManager::RemoveCodeHeapFromDomainList(CodeHeap *pHeap, LoaderAllocator
     }
 }
 
-void EEJitManager::FreeCodeMemory(HostCodeHeap *pCodeHeap, void * codeStart)
+#ifdef FEATURE_INTERPRETER
+
+InterpreterJitManager::InterpreterJitManager() :
+    m_interpreter(NULL),
+    m_interpreterHandle(NULL),
+    m_interpreterLoadCritSec(CrstSingleUseLock)
+{
+    LIMITED_METHOD_CONTRACT;
+    SetCodeManager(ExecutionManager::GetInterpreterCodeManager());
+}
+
+BOOL InterpreterJitManager::LoadInterpreter()
+{
+    STANDARD_VM_CONTRACT;
+
+    // If the JIT is already loaded, don't take the lock.
```

Comment:
nitpick: we say 'jit' and 'interpreter' interchangeably a few times in this function, i assume because it's based on altjit loader logic

Reply:
That's a copy & paste issue, I'll fix it. This code is a clone of the `EEJitManager::LoadJIT`